supdup.c

```diff
@@ -84,12 +85,12 @@ extern char *tgetstr();
 
 static char tspace[2048], *aoftspace;
 
-#define OUTSTRING_BUFSIZ 2048
-unsigned char *outstring;
-
 unsigned char *tparam(), *tgoto();
 #endif /* TERMCAP */
 
+#define OUTSTRING_BUFSIZ 2048
+unsigned char *outstring;
+
```

Comment:
Because it was used in both the TERMCAP and the TERMINFO branches, but only declared in the TERMCAP branch